templates/webapps/galaxy/workflow/run.mako

```diff
@@ -576,15 +576,14 @@ if wf_parms:
 
 %if has_upgrade_messages:
 <div class="warningmessage">
-    Problems were encountered when loading this workflow, likely due to tool
-    version changes. Missing parameter values have been replaced with default.
-    Please review the parameter values below.
+    Warning: Some tools in this workflow have been updated. The workflow may still work, but any new inputs will have default values.
```

Comment:
You're still using "parameter value" in the line below and I think the word is used pretty consistently in the rest of Galaxy docs and UI, so I am not sure it makes sense to change it just here.